Assets/Scripts/Enemy.cs

```diff
@@ -2,5 +2,6 @@
 
 public class Enemy : Character
 {
-
-}
+    public int shield;
+    public ElementType[] weakElement;
```

Comment:
```suggestion
    public ElementType[] weakElements;
```

배열은 복수형태로 바꾸는게 좋을 듯 해요
그래야지 forEach나 for문 돌릴 때 각 아이템의 변수를 정확하게 정의할 수 있어요
그리고 나머지 일부 배열의 변수명도 단수로 되어 있는 것 같아서 확인 하면 좋을 것 같아요